modin/engines/ray/utils.py

```diff
@@ -153,18 +153,27 @@ def initialize_ray(
                     object_store_memory = None
             else:
                 object_store_memory = int(object_store_memory)
-            ray.init(
-                num_cpus=CpuCount.get(),
-                include_dashboard=False,
-                ignore_reinit_error=True,
-                _plasma_directory=plasma_directory,
-                object_store_memory=object_store_memory,
-                address=redis_address,
-                _redis_password=redis_password,
-                logging_level=100,
-                _memory=object_store_memory,
-                _lru_evict=True,
-            )
+
+            ray_init_kwargs = {
+                "num_cpus": CpuCount.get(),
+                "include_dashboard": False,
+                "ignore_reinit_error": True,
+                "_plasma_directory": plasma_directory,
+                "object_store_memory": object_store_memory,
+                "address": redis_address,
+                "_redis_password": redis_password,
+                "logging_level": 100,
+                "_memory": object_store_memory,
+                "_lru_evict": True,
+            }
+            try:
+                ray.init(**ray_init_kwargs)
+            except DeprecationWarning:
+                # setting of `_lru_evict` parameter raises DeprecationWarning
+                # since ray 2.0.0
+                ray_init_kwargs.pop("_lru_evict")
+                ray.init(**ray_init_kwargs)
```

Comment:
This looks really weird to check a parameter like that - by doing multiple initializations....
I think a better way would be to do smth like (untested!)

```suggestion
            from packaging import version
            # setting of `_lru_evict` parameter raises DeprecationWarning since ray 2.0.0
            if version.parse(ray.__version__) >= version.parse('2.0.0'):
                ray_init_kwargs.pop("_lru_evict")
            ray.init(**ray_init_kwargs)
```